snap/snapcraft.yaml

```diff
@@ -43,5 +41,29 @@ parts:
     flutter-target: lib/main.dart
     build-packages:
       - libunwind-dev
-      - libgstreamer1.0-dev
-      - libgstreamer-plugins-base1.0-dev
+      - libmpv-dev
+  deps:
+    plugin: nil
+    stage-packages:
+      - libtheora0
+      - libunwind8
+      - libmpv-dev
```

Comment:
I have no experience in GNU/Linux packaging formats. Please take everything I say with a grain of salt.

<hr>

https://github.com/ozmartian/vidcutter/blob/8f01c76f0ec727fa336cb2cb6a645a58e3a29e64/_packaging/snap/snapcraft.yaml#L108

It seems `libmpv-dev` is the development/build package.
`libmpv1` is dependency runtime package (does not include mpv) containing the libmpv shared library.

One thing I noticed the other day is that this `libmpv1` package is `libmpv2` in Debian unstable.

So, I don't know what's the convention, does one ship separate package for two versions of Debian/Ubuntu? On the other hand, if one depends on `libmpv-dev` (& `mpv` ?), it automatically transitively depends on one of `libmpv1` or `libmpv2`, & confusion between `libmpv1` & `libmpv2` is eliminated, but users also get `mpv` installed. 


Regardless, if Snap allows to bundle the shared library within package like Flatpak, then you can strip down a ton of things from libmpv to make it only play audio. On Windows, I reduced size from 70 MB to 15 MB!!!

<hr>

Learn more: https://github.com/alexmercerind/mpv-win32-cmake/tree/disable-gl

Ideally, I want to offer separate self-sufficient audio & video libs packages for GNU/Linux in [`media_kit`](https://github.com/alexmercerind/media_kit) like on Windows. That'd include building & statically linking FFmpeg & mpv etc. etc. However, I believe this is against GNU/Linux philosophy (?).

Reply:
Thanks for the input @alexmercerind !